src/BuildPrediction/Microsoft.Build.Prediction.csproj

```diff
@@ -8,6 +8,7 @@
     <PackageReference Include="Microsoft.Build.Framework" Version="16.3.0" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.3.0" />
     <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="0.4.1" PrivateAssets="all" />
+    <PackageReference Include="System.Text.Json" Version="4.7.0" />
```

Comment:
I decided to use this instead of Newtonsoft.Json since this is the "new hotness" and supposedly faster at deserialization.